.github/workflows/publish.yml

```diff
@@ -68,6 +68,7 @@ jobs:
             json-rpc/logs-provider,
             json-rpc/polkadot-sdk-compat,
             json-rpc/sm-provider,
+            json-rpc/ws-middleware,
```

Comment:
For a moment I was like "won't this break main builds"? But no, this gets merged in v2 😅